audio-analysis/main.py

```diff
@@ -37,6 +39,11 @@ def analyzeVideoSound(url, tag):
         else:
             print('Join failed ... each word does not have an amplitude')
         print(df3)
+        df3.to_csv("data_export.csv", header=True)
+
+        engine = create_engine('mysql+pymysql://root:root@localhost/climactic_test')
+        df3.to_sql("test_table", con=engine, if_exists='replace')
+
```

Comment:
So for anyone looking at this in the future - the first "root" in _root:root@localhost/climactic_test_ is the username and the second is the password. You might have to remove the root after the colon if your localhost/server does not require a password.

Reply:
> @tjlam yep ... This is just hello world that overwrites. @Eldrick19 is writing the dedupe logic that will append to table.

Nice, would it make sense to create a unique key to help with deduping? maybe a combo of youtube url and word number?